.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: minimal
+
+sudo: false
+
+env:
+  global:
+    - secure: 
```

Comment:
Don't merge this yet. I need to get Travis-CI configure first and then use `doctr` to create the encrypted key.